addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -10,13 +10,18 @@ Currently, the binding supports a single type of Thing, being the ```command```
 
 The binding does not require any specific configuration.
 
-Note that the commands are executed in the context and with the privileges of the process running the java virtual machine.
-It is not advised to run the virtual machine as superuser/root.
+
+Note that the commands are executed in the context and with the privileges of the process running the java virtual machine. It is not advised to run the virtual machine as superuser/root. Linux Os needs the user openhab/openhabian to be able to execute dedicated command, it is advised to always test in the command line if this is possible.
+
+```
+sudo -u openhab <YOUR COMMAND>
+```
 
 ## Thing Configuration
 
-The `command` Thing requires the command to execute on the shell.
-Optionally one can specify:
+
+The ```command``` Thing requires the command to execute on the shell. Optionally one can specify:
```

Comment:
As @martinvw said, these extra backticks are not needed. In fact I'd write "command" Thing rather than `command` Thing.

Please do not merge sentences but keep them on separate lines. Markdown will merge them in the produced html pages automatically.